tests/LogTargetTest.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace yiiunit\extensions\debug;
+
+use yii\debug\LogTarget;
+use yii\debug\Module;
+
+class LogTargetTest extends TestCase
+{
+    public function testGetRequestTime()
+    {
+        $logTarget = new LogTarget(new Module('debug'));
+        $actual = $this->invoke($logTarget, 'getRequestTime');
+        $this->assertTrue(is_float($actual));
```

Comment:
Test does not seem to be enough.
Better to mock `$_SERVER['REQUEST_TIME_FLOAT']` value here and verify if it is actually picked up.